src/components/UI/CardsStyled/Cards.styled.js

```diff
@@ -2,36 +2,40 @@ import styled from 'styled-components';
 
```

Comment:
I think you are supposed to have one file per styled component (I think this is what Dominik told us during the first capstone week), but I'm not 100% sure, maybe ask him about it.